plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -9,13 +9,14 @@ describe('TaskTableUtil', function () {
   beforeEach(function () {
     this.getServiceHealth = MarathonStore.getServiceHealth;
     MarathonStore.getServiceHealth = function (prop) {
-      return this[prop].get('health');
+      return this[prop]['health'];
```

Comment:
The item should be of type `Item`. Which includes the get function so this line is fixing the issue but not the problem. Let's rather try to find the source of the problem. And ensure that all tasks are of type `Item`